base/special/rem_pio2.jl

```diff
@@ -208,12 +208,13 @@ function paynehanek(x::Float64)
 end
 
 """
-    rem_pio2_kernel(x)
+    rem_pio2_kernel(x::Union{Float32, Float64})
 
-Return the remainder of `x` modulo π/2 as a double-double pair, along with a `k`
-such that ``k \\mod 3 == K \\mod 3`` where ``K*π/2 = x - rem``. Note, that it is
-only meant for use when ``|x|>=π/4``, and that ``π/2`` is always subtracted or
-added for ``π/4<|x|<=π/2`` instead of simply returning `x`.
+Calculate `x` divided by `π/2` accurately for arbitrarily large `x`.
+Returns a pair `(k, r)`, where `k` is the quadrant of the result
+(multiple of π/2) and `r` is the remainder, such that ``k * π/2 = x - r``.
```

Comment:
We could say here that `k` is conceptually `mod(div(x, pi/2), 4)` just in higher precision than the naïve calculations, unless `quadrant` is sufficiently clear to the reader (whoever that is)?